webapp/sources/rudder/rudder-web/src/main/elm/editor/sources/ViewBlock.elm

```diff
@@ -232,9 +232,10 @@ showBlockTab model parentId block uiInfo techniqueUi =
                                        Call _ c -> Maybe.withDefault (c.methodName.value) (Maybe.map .name (Dict.get c.methodName.value model.methods))
                                    else
                                      componentValue
+
                      in
                        element "li"
-                               |> addActionStopAndPrevent ("click", MethodCallModified (Block parentId {block | reportingLogic = FocusReport (getId child).value }))
```

Comment:
That should still be the id. The id needs to be used in the focus report, not the component name.

You should change the display